easybuild/easyconfigs/s/STAR/STAR-2.5.3a-foss-2019b.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'MakeCp'
+
+name = 'STAR'
+version = '2.5.3a'
+
+homepage = 'https://github.com/alexdobin/STAR'
+description = "STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Comment:
I don't think there is a MPI or BLAS dep here, so it should be dropped to GCC level.